lib/rubocop/cop/mixin/statement_modifier.rb

```diff
@@ -4,6 +4,8 @@ module RuboCop
   module Cop
     # Common functionality for modifier cops.
     module StatementModifier
+      include LineLengthHelp
```

Comment:
I'm not super happy about mixing in another module here, polluting the namespace, so to speak. I'm open to suggestions.